internal/db/db.go

```diff
@@ -0,0 +1,89 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3" // Импорт драйвера SQLite
+	"log"
+	"os"
+)
+
+const dbFileName = "scheduler.db"
+
+// DB представляет собой обёртку для работы с базой данных
+type DB struct {
+	DB *sql.DB // Экспортируемое поле
+}
+
+// NewDB создаёт новое подключение к базе данных и возвращает экземпляр DB
+func NewDB() (*DB, error) {
+	// Получаем путь к исполняемому файлу
+	dbFile, err := os.Executable()
+	if err != nil {
+		return nil, fmt.Errorf("ошибка при получении пути к приложению: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return nil, fmt.Errorf("ошибка при получении пути к приложению: %w", err)
```